lib/gemstash/version.rb

```diff
@@ -2,5 +2,5 @@
 
 #:nodoc:
 module Gemstash
-  VERSION = "2.1.0"
+  VERSION = "2.1.1"
```

Comment:
Can you revert this change? We didn't decide to release 2.1.1 with this change yet.

Reply:
sure, I just updated the PR :)

Reply:
Thanks!